mattermost-plugin/.golangci.yml

```diff
@@ -67,7 +67,6 @@ linters:
     - unconvert
     - unused
     - whitespace
-    - gocyclo
```

Comment:
Why are we removing this?

Reply:
@jespino it was complaining about cyclomatic complexity exceeding the default threshold of 30 for the data migration function (which currently handles about 4-5 kinds of data migrations). IN the future, I see only more data migrations being added there, so increasing the linter's threshold value won't work in the long term.

I can optimize the code to reduce the conditions and returns paths but it won't work for long when we add the next data migration.

Hence, I removed the linter altogether. I'm open to any alternate solution.